src/select/__tests__/__snapshots__/Select.test.tsx.snap

```diff
@@ -14,6 +14,7 @@ exports[`Select should render correctly 1`] = `
       Select a fruit
     </button>
     <div
+      aria-activedescendant="select-apple"
```

Comment:
Why apple is already in the activedescendant without providing the "selectedValue"?